tools/certa/package.json

```diff
@@ -18,7 +18,8 @@
     "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
     "test": "",
     "docs": "",
-    "cover": ""
+    "cover": "",
+    "postinstall": "node ./installPlaywright.js"
```

Comment:
i know this is what we did in the auth clients repo, but hoping theres a better way

Reply:
looking into alternative